### PR TITLE
Invoice repo

### PR DESCRIPTION
**Pull request by avjohnston**
Finished the invoice_repo and invoice_repo_test, if it looks good we can move on to integrating invoice to our sales_analyst.

### CODE REVIEW

**Review by Diana20920 — APPROVED** [review]
This is looking more and more awesome! 🔥

File: test/invoice_test.rb [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+      @attributes = {
+        :id          => '6',
+        :customer_id => '7',
+        :merchant_id => '8',
         :status      => "pending",
-        :created_at  => "2015-03-13", #Time.now,
-        :updated_at  => "2015-04-05" #Time.now,
-      }, "self")
+        :created_at  => "2015-03-13"
+        :updated_at  => "2015-04-05"
```
[the remark this answers is not in the document]
Comment (brisag) [reply test/invoice_test.rb:17]: TIme.now gave milisecond changes, glas we are using stings from the csv files

**Comment by DougWelchons** [comment]
Looks like everything is tracking with where we wanted to be.

**Comment by brisag** [comment]
Looks Fantastic! Glad it passes spec harness

**Review by DougWelchons — APPROVED** [review]

**Pull request merged**